src/processors/controllers/processorController.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+let processorManager = require('../models/processorsManager');
+
+module.exports.createProcessor = function (req, res, next) {
+    return processorManager.createProcessor(req.body)
+        .then(function (result) {
+            return res.status(201).json(result);
```

Comment:
the response missing created_at/updated_at

Reply:
done